Backend/src/routes/api.js

```diff
@@ -100,200 +103,73 @@ ROUTER.post("/record-data", (req, res) => {
   console.log("Record Status:", req.body)
 });
 
-ROUTER.post("/get-recorded-data/x", async (req, res) => {
+
+ROUTER.post("/process-recorded-data/x", async (req, res) => {
   currentSession = req.body.fileName
   if (currentSession === "") {
     res.send({ response: "NoFile" }).status(200)
     return
   }
 
-  console.log("Sess:", req.body)
-  // await getrecordedData()
-  // res.send({ response: RecodedData }).status(200)
+  res.send({ response: currentSession}).status(200)
 });
 
 
-ROUTER.get("/get-recorded-data", (req, res) => {
+ROUTER.get("/process-recorded-data", (req, res) => {
   // Execute Python script to convert recorded binary data to a formatted Excel file
 
-  var dataToSend = "";
-  // spawn new child process to call the python script
-  //const python = spawn('python', ['C:\\Users\\james\\badgerloop_repo\\chase-car-dashboard\\Backend\\src\\routes\\script1.py']);
-
-  // TODO Add installing python/pip (and any dependencies/additional modules, like `pip install numpy` and `pip install (--user most likely?) XlsxWriter`) in terminal in which npm is used to README
+  console.log(currentSession)
 
-  console.log(currentSession);
+  if (currentSession === "") {
+    res.send({ response: "NoFileSelected" }).status(200);
+    return
+  }
 
-  // TODO const python = spawn('python', ['./src/routes/exp_new_script1.py','./src/routes/demo3.bin','./Data/sc1-data-format/format.json','./src/routes/test.csv']);
-  const python = spawn('python', ['./src/routes/exp_new_script1.py',
+  // Spawn new child process to call the python script
+  const python = spawn('python', ['./src/routes/process_recorded_data.py',
```

Comment:
we should check for python version here or enforce a dockerized enviornment here. python doesn't mean python for everyone else

Reply:
Yeah, the plan is to match the docker environment. Then, I can add some instructions to the README to alias `python` (or `python3` or `py` or whatever it happens to be) to refer to an appropriate version of Python 3.X.X.